test/util.test.ts

```diff
@@ -0,0 +1,54 @@
+import { promises } from 'fs';
+import { tmpdir } from 'os';
+import path from 'path';
+import { getFiles } from '../src/util';
+
+describe('getFiles', () => {
+
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test('test read files', async () => {
+
+    //WHEN we have this folder structure
+
+    // Only look for files of this type
+    const extensionFilter = 'k8s.yaml';
+    // Create folders
+    const baseDir: string = await promises.mkdtemp(path.join(tmpdir(), 'cdk8s-getFiles-test'));
+    const emptySubDir: string = await promises.mkdtemp(path.join(baseDir, 'emptyDir'));
+    const PopulatedSubDir: string = await promises.mkdtemp(path.join(baseDir, 'populatedDir'));
+    const PopulatedSubDir2: string = await promises.mkdtemp(path.join(PopulatedSubDir, 'populatedDir2'));
+    // Do not create a folder in this path
+    const nonDir: string = path.join(baseDir, 'nonDir');
+    //Create Files
+    await promises.writeFile(path.join(PopulatedSubDir, `example.${extensionFilter}`), 'contents');
+    await promises.writeFile(path.join(PopulatedSubDir, `example2.${extensionFilter}`), 'contents');
+    await promises.writeFile(path.join(PopulatedSubDir2, `example3.${extensionFilter}`), 'contents');
+    await promises.writeFile(path.join(PopulatedSubDir2, 'example4.txt'), 'contents');
+
+
+    // THEN
+
+    // Get the contents of the different folders
+    const noFiles : string[] = await getFiles(emptySubDir);
+    const yamlFiles : string[] = await getFiles(PopulatedSubDir);
+    const NoFilesEither : string[] = await getFiles(nonDir);
```

Comment:
style - no space between variable name and ":"
```suggestion
    const noFiles: string[] = await getFiles(emptySubDir);
    const yamlFiles: string[] = await getFiles(PopulatedSubDir);
    const noFilesEither: string[] = await getFiles(nonDir);
```